sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -612,8 +612,10 @@
 
     /**
      * Lists chronological/historical representation of {@link ConfigurationSetting} resource(s). Revisions are provided
-     * in descending order from their {@link ConfigurationSetting#getLastModified() lastModified} date. Revisions expire
-     * after a period of time. The service maintains change history for up to 7 days.
+     * in descending order from their {@link ConfigurationSetting#getLastModified() lastModified} date.
+     * Revisions expire after a period of time.
+     * <p>See <a href="https://azure.microsoft.com/en-us/pricing/details/app-configuration/">Pricing</a>
+     * for more information.</p>
```

Comment:
Why does this link to pricing? Is there a better page of documentation which discusses how long a revision will be maintained?

Reply:
key History in pricing. Price = free, 7 days; Price = standard, 30 days. Keep it as link as we don't want to set it to fixed.